roles/prometheus/tasks/cleanup.yml

```diff
@@ -1,8 +1,15 @@
 # code: language=ansible
 ---
+- name: Check the status of firewalld
```

Comment:
Cannot we run this in the `playbook.yaml` file?
I mean, instead of repeating 4 times

Reply:
This is probably due to my very limited Ansible knowledge, but I don't know how: it needs to be in two different roles, right (because on each machine that is executing each role)? so that means I need to put it in at least two plays in the playbook. And I would also need to specify it another two times for the cleanup.yml, right?